### DISCUSSION TITLE
How i build build.py rustdesk on window...

### DISCUSSION THREAD

**Discussion by minkik715** [rustdesk/rustdesk#5238] [General]
I take a long time to build rustdesk...

1. git clone https://github.com/rustdesk/rustdesk.git  and check out branch or version you want
2.  Follow this instruction https://rustdesk.com/docs/en/dev/build/windows/ (when i install aom:x64-windows-static, It took about 30 minutes, so please wait long enough)(if you met some error check here https://rustdesk.com/docs/en/dev/build/faq/ or write comment) 
3. i remeber success to cargo build, but i met old version ui, and i have to put src/ui file and scitter.dll
4. after searching, i realize that i have to build build.py  
5. to build build.py we have to install python, flutter, dart, android studio, android sdk, and check flutter doctor work well.
6. we also need to bridge generated file so use these command on powershell
 cargo install flutter_rust_bridge_codegen --version 1.75.3 --features uuid
 flutter pub get
 ~/.cargo/bin/flutter_rust_bridge_codegen --rust-input ../src/flutter_ffi.rs --llvm-path **C:/dev/LLVM(your llvm path)** --dart-output ./lib/generated_bridge.dart 
7. change python3 -> python in build.py file
8. python build.py --flutter 
9. i can get one file to install or execute rustdesk.. 
Good Luck..

**Comment by it-boerner** [comment]
Were you also able to make adjustments in your own RustDesk client?

Were you also able to make adjustments to the design etc. or even create a QuickSupport client?

**Reply by minkik715** [reply]
I modified core_main.rs as I wanted.
I think you can also change some code as you want.

**Comment by MarkoCuk27** [comment]
Hello,  I have an issue with the ffigen package. Did you have similar problems?

**Reply by urani-dev** [reply]
@MarkoCuk27 you should use correct version of flutter-rust-bridge as setting here: https://github.com/rustdesk/rustdesk/blob/master/.github/workflows/flutter-build.yml#L29

**Comment by xubingyan** [comment]
When I run flutter_rust_bridge_codegen ，[SEVERE] : Couldn't find dynamic library in default locations.
[SEVERE] : Please supply one or more path/to/llvm in ffigen's config under the key 'llvm-path'.
